Dockerfile

```diff
@@ -0,0 +1,13 @@
+# http://phusion.github.io/baseimage-docker/
+FROM phusion/baseimage
+
+RUN  add-apt-repository ppa:webupd8team/java && \
+     add-apt-repository -y ppa:webupd8team/java && \
+     echo debconf shared/accepted-oracle-license-v1-1 select true | sudo debconf-set-selections && \
```

Comment:
It should be, this is completely orthogonal to travis.  These commands are run in a docker vm in order to set up the docker image.